modules/ROOT/pages/find-conformance-issues.adoc

```diff
@@ -10,59 +10,27 @@ You can check for API specification conformance using a custom or MuleSoft-provi
 
 See xref:find-and-fix-conformance-issues.adoc[] for an overview of ways to validate conformance and determine why conformance failed.
 
-[[validate-in-design-center]]
-== Validate API Specifications in Design Center
+[[validate-specs]]
+== Validate API Specifications Against Rulesets
 
-If your API is not governed using API Governance or if you want to see conformance issues directly in API Designer, you can validate your API in API Designer. 
+You can validate APIs against rulesets within the context of your API project regardless of whether your API is centrally governed through Anypoint API Governance. Developers or implementors typically do this either to proactively identify conformance issues or to review issues they're notified about.
```

Comment:
If I understand your concern correctly, we don't qualify the aspect of the API every time we mention it throughout the doc, and rely on the context quite a bit. Here is the definition we have now for [API](https://docs.mulesoft.com/general/glossary#a:~:text=application%20network%20graph.-,API,-The%20entire%20API) (I also reuse this in the API Gov doc). It would be a pretty huge impact to the doc to qualify everywhere, but certainly if this is a place where we really should, I can.

Reply:
Perhaps I should add Anypoint Code Builder into that definition for visibility, now that I think about it, even though those are just examples.

Reply:
Yeah, in ACB land, we think about designing an API spec and for governance, running rules against the spec. Then we talk about implementing a scaffolded API interface as a Mule app.